sources/packages/backend/workflow/test/2022-2023/parttime-assessment/appeals/parttime-assessment-student-additional-transportation.e2e-spec.ts

```diff
@@ -0,0 +1,182 @@
+          additionalTransportKm: 12,
+          additionalTransportCost: 111,
+          additionalTransportWeeks: 1,
+          transportationCostSituation: TransportationCostSituation.NoLimit,
+          additionalTransportPlacement: YesNoOptions.Yes,
+        };
+      // Act
+      const calculatedAssessment =
+        await executePartTimeAssessmentForProgramYear(
+          PROGRAM_YEAR,
+          assessmentConsolidatedData,
+        );
+
+      // Assert
+      expect(
+        calculatedAssessment.variables.calculatedDataAdditionalTransportKm,
+      ).toBe(12);
+      expect(
+        calculatedAssessment.variables.calculatedDataAdditionalTransportCost,
+      ).toBe(111);
+      expect(
+        calculatedAssessment.variables.calculatedDataAdditionalTransportWeeks,
+      ).toBe(1);
+      expect(
+        calculatedAssessment.variables
+          .calculatedDataAdditionalTransportPlacement,
+      ).toBe(true);
+    },
+  );
+
+  it(
+    "Should have calculated student additional transportation variables assigned with request a change values " +
+      "when there is a request a change for student additional transportation with minimal required fields defined.",
+    async () => {
+      // Arrange
+      const assessmentConsolidatedData =
+        createFakeConsolidatedPartTimeData(PROGRAM_YEAR);
+      assessmentConsolidatedData.studentDataAdditionalTransportKm = 30;
+      assessmentConsolidatedData.offeringWeeks = 20;
+      assessmentConsolidatedData.studentDataAdditionalTransportWeeks = 15;
+      assessmentConsolidatedData.studentDataAdditionalTransportCost = 30;
+      assessmentConsolidatedData.studentDataAdditionalTransportPlacement =
+        YesNoOptions.No;
+
+      assessmentConsolidatedData.appealsStudentAdditionalTransportationAppealData =
+        {
+          eligibleForAnAdditionalTransportationAllowance: YesNoOptions.Yes,
+          additionalTransportKm: 12,
+          additionalTransportCost: 111,
+          transportationCostSituation: TransportationCostSituation.NoLimit,
+          additionalTransportPlacement: YesNoOptions.Yes,
+        };
+      // Act
+      const calculatedAssessment =
+        await executePartTimeAssessmentForProgramYear(
+          PROGRAM_YEAR,
+          assessmentConsolidatedData,
+        );
+
+      // Assert
+      expect(
+        calculatedAssessment.variables.calculatedDataAdditionalTransportKm,
+      ).toBe(12);
+      expect(
+        calculatedAssessment.variables.calculatedDataAdditionalTransportCost,
+      ).toBe(111);
+      expect(
+        calculatedAssessment.variables.calculatedDataAdditionalTransportWeeks,
+      ).toBe(0);
+      expect(
+        calculatedAssessment.variables
+          .calculatedDataAdditionalTransportPlacement,
+      ).toBe(true);
+    },
+  );
+
+  it(
+    "Should have calculated student additional transportation variables assigned with 0 " +
+      "when there is a request a change for student additional transportation not required.",
```

Comment:
Was the "not required" kept from the previous implementation or because it is still meaningful?
Should we update to something that would mention "not eligible for transportation allowance"?

Reply:
Changed to:  
"Should have calculated student additional transportation variables assigned with 0 " +
      "when there is a request a change for not eligible for transportation allowance.",

Reply:
I believe that `2022-2023` file is missing the change.

Reply:
Fixed.